scripts/train.py

```diff
@@ -160,7 +160,7 @@ def main():
         torch_dtype=POLICY_MODEL_PRECISION,
     )
     tokenizer = AutoTokenizer.from_pretrained(model_name)
-    tokenizer.pad_token = tokenizer.eos_token
+    # tokenizer.pad_token = tokenizer.eos_token
```

Comment:
Curious about the thought behind this, when I run it it sends a message `Setting `pad_token_id` to `eos_token_id`:151643 for open-end generation.` so I'm not sure how much it changes but I want to hear what you're thinking.

Reply:
when looking at the end of the responses of the model, they usually have the EOS token and bunch of padding tokens. However, by setting the padding token to the EOS token, if we zero out the padding tokens, we also zero out the actual EOS tokens which we want to keep.

Reply:
Makes sense, thanks for making the change.

Reply:
Not a big deal but we can probably just delete the line then
```suggestion
    tokenizer.padding_side = "left"
```

Reply:
Sounds good, will update it